wiki/Beatmap/Overall_difficulty/en.md

```diff
@@ -30,6 +30,10 @@ Hit windows for judgments can be viewed when hovering over the [beatmap informat
 | 100 | `140 - 8 * OD` |
 | 50 | `200 - 10 * OD` |
 
+#### Table comparison
+
+![](/wiki/shared/ODTable.png "Comparison of hit windows for different combinations of OD and game modifiers")
```

Comment:
the OD values for DT/HT shouldn't be kept without explanation -- or maybe at all

the numbers for OD there are saying "a map with this OD on nomod would have the same hit window for 300s", but it would be different for 100 and 50, so it's kind of misleading.

Reply:
"but it would be different for 100 and 50" -- what do you mean here? could you coin a full explanation, perhaps?

Reply:
given an OD you can find hit windows using the formulas above, but given the time-scaled hit windows from DT or HT, you can't map them back to a single OD value

for example look at OD5+DT
300 window is 33.33ms = nomod OD7.78 (this is shown on the chart)
100 window is 66.67ms = nomod OD9.17
50 window is 100ms = nomod OD10

when i made the chart, I put the equivalent nomod OD for the 300 hit window, since that's probably what most players care to see. but without explanation it can be misleading

Reply:
thanks, that makes it clear. does this work? https://github.com/ppy/osu-wiki/pull/8144/commits/38a8843588ca147b28f12cfd3c3c04e53521f28b